neofetch

```diff
@@ -1709,7 +1709,11 @@ get_battery() {
 
         "Mac OS X")
             battery="$(pmset -g batt | grep -o '[0-9]*%')"
-            battery_state="$(pmset -g batt | awk 'NR==2 {print $3}')"
+            state="$(pmset -g batt | awk '/;/ {print $4}')"
+            if [ "$state" == "charging;" ];
```

Comment:
Could this be changed to a one-line if statement please? This would like like this:

```bash
[[ "$state" == "charging;" ]] && battery_state="charging"
```